provider/server/machine_volume_detach_test.go

```diff
@@ -0,0 +1,165 @@
+// SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and IronCore contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package server_test
+
+import (
+	"time"
+
+	"github.com/digitalocean/go-libvirt"
+	iri "github.com/ironcore-dev/ironcore/iri/apis/machine/v1alpha1"
+	irimeta "github.com/ironcore-dev/ironcore/iri/apis/meta/v1alpha1"
+	libvirtutils "github.com/ironcore-dev/libvirt-provider/pkg/libvirt/utils"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"libvirt.org/go/libvirtxml"
+)
+
+var _ = Describe("DetachVolume", func() {
+	It("should correctly detach volume from machine", func(ctx SpecContext) {
```

Comment:
Can we also cover the case of ceph volume detachment as well?

Reply:
Done